kong/core/reports.lua

```diff
@@ -99,8 +99,6 @@ local function send_report(signal_type, t, host, port)
 
   -- concat and send buffer
 
-  --print(concat(_buffer, ";", 1, mutable_idx))
```

Comment:
I believe we should have more easily debugging statements ready to be uncommented in the code, that is the reason why this one exists. If you are worried about the repeated statement, then please update it to:
```lua
local msg = concat(_buffer, ";", 1, mutable_idx)

--print(msg)

ok, err = sock:send(msg)
```

It's easy for us to "copy and paste" because we know how the code works. For someone who isn't familiar with the code base, it's easier to browse the code and see where are the debugging steps that can simply be un-commented.